### PR TITLE
Add some missing committers to the AUTHORS list

### LINKED ISSUE #13815: AUTHORS is out of date
Many project members are missing.

### PR DESCRIPTION
**Pull request by mspncp**
Fixes #13815

### CODE REVIEW

**Review by paulidale — APPROVED** [review]

**Review by levitte — APPROVED** [review]

**Comment by levitte** [comment]
I suspect most of the added names should be added in the 1.1.1 AUTHORS as well, no?

**Comment by paulidale** [comment]
I'm fine with them going into 1.1.1 too.

File: AUTHORS.md
Comment (t8m) [inline AUTHORS.md:26]: Dmitry Belyavskiy

Comment (mspncp) [reply AUTHORS.md:26]: Ok, thanks for the hint. I copied it from https://www.openssl.org/community/committers.html, it needs to be corrected there, too. I should have looked at https://github.com/beldmit. :-)

Comment (mspncp) [reply AUTHORS.md]: It looks like the spelling name is in a file `committers.inc`, which is [included from committers.html](https://github.com/openssl/web/blob/15c3d9188ef04d9d3d4b98088d641163390a5e03/community/committers.html#L26) and is not publicly visible.

Comment (mspncp) [reply AUTHORS.md]: Can you take care of this @t8m?

Comment (beldmit) [reply AUTHORS.md]: Oh... It's much longer story. I used both spellings.

May I ask to add something like Dmitry Belyavskiy (Dmitry Belyavsky, Дмитрий Белявский)?

Comment (mspncp) [reply AUTHORS.md]: Sure. But three Names is a little bit much IMHO, how about just adding your cyrillic name?

```
*  Dmitry Belyavskiy (Дмитрий Белявский)
```

Comment (mattcaswell) [reply AUTHORS.md]: I see no problem with 3 aliases if they're all valid. Its better to be clear rather than have any confusion.

Comment (mspncp) [reply AUTHORS.md]: Of course, if you prefer to have all three names listed, I'll do as you wish.

Comment (mspncp) [reply AUTHORS.md]: @mattcaswell I didn't see your reply before force-pushing. I'll wait for @beldmit's feedback and change it if he prefers the three names.

Comment (beldmit) [reply AUTHORS.md]: Dmitry Belyavskiy (Дмитрий Белявский) will be enough, I think. Many thanks!

Comment (mspncp) [reply AUTHORS.md]: Just for curiousity: why is the ий transcribed differently in your first and last name?

Comment (beldmit) [reply AUTHORS.md]: Some weir reasons, in fact. The rules of transliteration in Russia has changed many times, and my current version is something intermediate.

**Review by paulidale — APPROVED** [review]

**Comment by paulidale** [comment]
I don't see a point in resetting the 24 hour timer for the change.

**Comment by mspncp** [comment]
Merged to master (af403db090ee66715e81f0062d1ef614e8d921b5) and 1.1.1 (2d8109f5f8205ac247630f397582727b9682be38), thanks!

**Pull request closed without merging**
